src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -257,6 +257,12 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
         // second idam call is workaround for null pointer when hiding field in getIdamEmail callback
         CaseData.CaseDataBuilder dataBuilder = getSharedData(callbackParams);
 
+        if (caseData.getRespondent2SameLegalRepresentative() == YES) {
+            // Predicate: Def1 registered, Def 2 unregistered.
+            // This is required to ensure mutual exclusion in 1v2 same solicitor case.
+            dataBuilder.respondent2OrgRegistered(YES);
```

Comment:
Before we set `Respondent2OrgRegistered` to true, can we also check that respondent 1 is registered. Currently this question "Is the legal rep the same" is shown in the UI, when respondent 1 is not registered. @jeejavr raised  bug to fix this - but we should still implement the check in the backend to be sure

Reply:
i agree with this, good point raised

Reply:
sure, in case the respondent 1 is not registered, how would the application behave? do you want it to set it to true?

Reply:
you'd need to do a 
```
 if (caseData.getRespondent2SameLegalRepresentative() == YES && caseData.getRespondent1OrgRegistered == YES) {
  dataBuilder.respondent2OrgRegistered(YES);
}
```

Reply:
If respondent 1 is not registered, another predicate should catch this.
We should only set  Respondent2OrgRegistered to YES when:
- Respondent 1 Is represented + Registered
- The respondents solicitors are the same

This is just a cleanup exercise, so that we can asses the predicate and continue along Claim Issued, when its a 1v2 Same Solicitor scenario.

Currently in master, this is failing - because it thinks it should progress to Claim Issued + also Hand Offline, because respondent 2 org is not registered.. If we set Org 2 to registered, the predicate will only resolve as Claim Issued

Reply:
@HarryH96 thanks for the reply. I have updated the code based on the mentioned condition. please take a look. Cheers!